appveyor.yml

```diff
@@ -51,4 +51,4 @@ on_finish:
 - ps: #$blockRdp = $true; iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1'))
 
 on_success:

```

Comment:
Maybe this should be `deploy_script:`, otherwise it will publish when a PR has a successful build? But maybe `on_success: `works the same as  `deploy_script:`?

Reply:
Although, if it does try to publish on a successful PR build, the publish command will fail since the environment variable containing the API key is not passed for a PR.